src/lib/swapper/utils.ts

```diff
@@ -2,26 +2,32 @@ import type { SwapErrorRight } from '@shapeshiftoss/swapper'
 import { makeSwapErrorRight, SwapErrorType, SwapperName } from '@shapeshiftoss/swapper'
 import type { Result } from '@sniptt/monads'
 import { Err, Ok } from '@sniptt/monads'
-import type { AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios'
-import type { ISetupCache } from 'axios-cache-adapter'
-import { setupCache } from 'axios-cache-adapter'
+import Axios, { type AxiosInstance, type AxiosRequestConfig, type AxiosResponse } from 'axios'
+import { setupCache } from 'axios-cache-interceptor'
 import type { FeatureFlags } from 'state/slices/preferencesSlice/preferencesSlice'
 
 import { isCrossAccountTradeSupported } from '../../state/helpers'
 import { AsyncResultOf, isTruthy } from '../utils'
 
-const getRequestFilter = (cachedUrls: string[]) => (request: Request) =>
-  !cachedUrls.some(url => request.url.includes(url))
+const getRequestFilter = (cachedUrls: string[]) => (request: AxiosRequestConfig) =>
+  !cachedUrls.some(url => request.url?.includes(url))
 
-export const createCache = (maxAge: number, cachedUrls: string[]): ISetupCache => {
+export const createCache = (
+  maxAge: number,
+  cachedUrls: string[],
+  axiosConfig: AxiosRequestConfig,
+): AxiosInstance => {
   const filter = getRequestFilter(cachedUrls)
-  return setupCache({
-    maxAge,
-    exclude: { filter, query: false },
-    clearOnStale: true,
-    readOnError: false,
-    readHeaders: false,
+  const axiosInstance = Axios.create(axiosConfig)
```

Comment:
One instance per `createCache` call, vs. us passing an axios instance previously. Having this at function vs. module scope isn't a mistake.
Since `createCache` now has to have an axios instance created from its own scope (vs. it returning a cache instance previously that could *then* be used to augment instances), this avoids different services using the same axios instance and being rugged as their config would be wrong with a shared instance.